backend/packages/Upgrade/src/api/services/ExperimentService.ts

```diff
@@ -819,7 +856,11 @@ export class ExperimentService {
         const { experimentId, ...restDoc } = partitionDoc as any;
         return restDoc;
       });
-      return { ...experimentDoc, conditions: conditionDocToReturn as any, partitions: partitionDocToReturn as any };
+      const queryDocToReturn = 
+        !!queryDocs && queryDocs.map((queryDoc) => {
+        return queryDoc;
+      });
+      return { ...experimentDoc, conditions: conditionDocToReturn as any, partitions: partitionDocToReturn as any, queries: (queryDocToReturn[0] as any) || []};
```

Comment:
Why the `[0]`?

Reply:
On `queryDocs.map()` it was returning array of array [[{},{}]]. So, had to give the first indexed Doc.